server/session/handler_item_stack_request.go

```diff
@@ -343,31 +408,35 @@ func (h *ItemStackRequestHandler) verifySlot(slot protocol.StackRequestSlotInfo,
 	if id := item_id(i); id != clientID {
 		return fmt.Errorf("stack ID mismatch: client expected %v, but server had %v", clientID, id)
 	}
-	inventory, _ := s.invByID(int32(slot.ContainerID))
 
+	inv, _ := s.invByID(int32(slot.ContainerID))
```

Comment:
To elaborate on the below comment, move this bit up to before `h.resolveID` and check the error.

Reply:
The inventory returned by this function is used later in resolveID though. Do you want me to pass inventory as a parameter to resolveID instead of the session?

Reply:
https://github.com/df-mc/dragonfly/blob/dd4c13056f4453d11f7055cc2af78fabf2a8f727/server/session/handler_item_stack_request.go#L439

Reply:
Yeah I think that'd be the way to go.